frontend/packages/dev-console/src/components/HintBlock.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { PageHeading } from '@console/internal/components/utils';
+import FormSectionDivider from './import/section/FormSectionDivider';
+interface HintBlockProps {
+  title: string;
+  children?: React.ReactNode;
+}
+const HintBlock: React.FC<HintBlockProps> = (props) => (
+  <React.Fragment>
+    <PageHeading title={props.title}>{props.children}</PageHeading>
+    <FormSectionDivider/>
```

Comment:
let's give this component a more generic name. This is being used outside of Forms also. Maybe `SectionDivider`